api/test/service/store.test.js

```diff
@@ -98,11 +98,9 @@ test('store/add returns signed url for uploading', async (t) => {
 
   const item = await getItemFromStoreTable(t.context.dynamoClient, tableName, spaceDid, link)
   t.truthy(item)
-  t.is(typeof item?.uploadedAt, 'string')
-  t.is(typeof item?.proof, 'string')
```

Comment:
why removing invocation?

Reply:
Fixed and now tests that `invocation` is a CID, and that `insertedAt` is a recent parseable Date!